hedera-node/src/main/java/com/hedera/services/context/primitives/StateView.java

```diff
@@ -507,6 +508,10 @@ public StakingInfo stakingInfo(final MerkleAccount account) {
 
 		if (account.getStakePeriodStart() > 0) {
 			stakingInfo.setStakePeriodStart(Timestamp.newBuilder().setSeconds(account.getStakePeriodStart()).build());
+			if (account.getStakedId() < 0 && !account.isDeclinedReward()) {
```

Comment:
NIT: could make this condition a `account.mayHavePendingReward()` method.

Reply:
Changed